src/pages/index.js

```diff
@@ -30,8 +111,17 @@ function fillProfileForm(e) {
 const profileEditForm = new PopupWithForm(
   "#profile-edit-popup",
   (inputValues) => {
-    pageUserInfo.setUserInfo(inputValues);
-    profileEditForm.close();
+    profileEditForm.setLoadingState(true);
+    api.updateUserInfo(inputValues).then(() => {
+      api
+        .getUserInfo()
+        .then((data) => {
+          profileEditForm.close();
+          pageUserInfo.setUserInfo(data);
+        })
+        .catch((err) => console.log(err))
```

Comment:
Needs correction: please move `catch` and `finally` blocks to `api.updateUserInfo` chain (line 124)

Reply:
corrected